web/src/routes/LicensePage.tsx

```diff
@@ -21,26 +21,21 @@ type PropsType = {
   onPress: string
 }
 
-const LicenseItem = (props: PropsType): ReactElement => {
-  const { name, version, license, onPress } = props
-  return (
-    <ListItem path={onPress} title={name}>
-      <LicenseContainer>
-        {`version: ${version}`}
-        <br />
-        {`license: ${license}`}
-      </LicenseContainer>
-    </ListItem>
-  )
-}
+const LicenseItem = ({ license, name, onPress, version }: PropsType): ReactElement => (
+  <ListItem path={onPress} title={name}>
+    <LicenseContainer>
+      <div>{`version: ${version}`}</div>
+      <div>{`license: ${license}`}</div>
```

Comment:
The following should work as well (maybe divs/br tag are needed again):
```suggestion
      version: {version}
      license: ${license}
```

Reply:
Well, I think both ways are ok and therefore I leave it as it is

Reply:
Well, please try to keep stuff as simple as possible. It just does not make sense to wrap the whole string in a template string  and use one template in another one if it works otherwise as well.

Reply:
I'll just fix it along with the other issues